pyproject.toml

```diff
@@ -18,31 +24,55 @@ classifiers = [
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
     "Topic :: Scientific/Engineering :: Astronomy",
     "Topic :: Scientific/Engineering :: Physics"
 ]
-
+dynamic = ["version"]
+requires-python = ">=3.9.0,<3.13"
 packages = [
     { include = "tom_alertstreams" }
 ]
+dependencies = [
+    "tomtoolkit >=2.22,<3.0",
+    "psycopg2-binary >=2.9,<3.0",
```

Comment:
I don't think `tom-alertstreams` should depend on `psycopg` (It doesn't have any models). If this is because of `tom_alertstreams_base`, then `tom_alertstreams_base` should probably be removed because now we have `make-tom.sh` to facilitate developement and testing.

Reply:
This feels like it's outside the scope of this PR.
I will make another issue.